internal/service/appautoscaling/scheduled_action.go

```diff
@@ -112,12 +112,16 @@ func ResourceScheduledAction() *schema.Resource {
 func resourceScheduledActionPut(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	var diags diag.Diagnostics
 	conn := meta.(*conns.AWSClient).AppAutoScalingConn(ctx)
+	currentStartTime, _ := time.Parse(time.RFC3339, d.Get("start_time").(string))
+	currentEndTime, _ := time.Parse(time.RFC3339, d.Get("end_time").(string))
```

Comment:
To avoid missing `StartTime` and `EndTime` whenever we try to update existing `ScheduledAction` I tried to set existing value, but after this, use might need to explicitly set `StartTime` or `EndTime` to `""` or `null` to remove the value.